### DISCUSSION TITLE
Allow a "Respawn as Monster" option for hosts

### DISCUSSION THREAD

**Discussion by kicknbrit** [FakeFishGames/Barotrauma#4929] [Ideas and suggestions]
Currently I feel the biggest issue with multiplayer is the sometimes slow pace of missions, especially in campaign mode.
I always see so many people trying to join games but they sit in lobby and wait and wait sometimes for 20+ minutes and finally leave. It would be nice to have dead/new players spawn as monsters to increase the difficulty and encourage crew members to keep a healthy pace.

Additionally, It may be fairly easy to implement seeing as how you only need a small command in-game for individual players. 
I am actually considering just modifying the server source and adding it in myself but I wanted to stop by here because I think the base game would greatly benefit from this.

Thanks for your time

**Comment by Leonidas-ai** [comment]
Just modify the source. There's already a command to spawn monsters and allow players to control them in vanilla. It's trivially simple to make your own custom command that combines these two commands to simplify the process.

**Reply by kicknbrit** [reply]
Well, its not "trivially simple" I don't know how familiar you are with the server code but if you take a look at it you would see what I mean. Also when you spawn as monsters you currently have no way to find the subs. I was hoping they could handle this because it would allow spectators to find subs instead of just wandering endlessly.

**Comment by mLuby** [comment]
@kicknbrit you might consider changing the title of this suggestion to `Add "spectator can control a monster" server option` since currently could be read to mean only server hosts would be able to do it, and it doesn't really have to do with respawning.

**Comment by kicknbrit** [comment]
Has any admin seen this post? Any thoughts?
